git_hg_sync/repo_synchronizer.py

```diff
@@ -91,7 +91,11 @@ def sync(
                 raise RepoSyncError(branch_operation, e) from e
 
         os.environ[REQUEST_USER_ENV_VAR] = request_user
-        logger.debug(f"{REQUEST_USER_ENV_VAR} set to {request_user}")
+        os.environ["GIT_AUTHOR_EMAIL"] = request_user
+        # We don't have the author name in the Pulse message, so we reuse the email
+        # address.
+        os.environ["GIT_AUTHOR_NAME"] = request_user
+        logger.debug(f"{REQUEST_USER_ENV_VAR} and GIT_AUTHOR_* set to {request_user}")
```

Comment:
Is this going to result in something like `name@example.org <name@example.org>`? If so, we should maybe take only the part before the `@` sign for the name to make things a little cleaner - though still not ideal. We should probably be passing the name and email in the original request so we can have all the needed info.

Reply:
Done. 

Yeah, unlike commits, pushes are only associated to an email address (requester_email), so we'd have to align it a bit more.